osmo_jupyter/timezone.py

```diff
@@ -0,0 +1,51 @@
+import datetime
+
+import pytz
+import pandas as pd
+
+
+OSMO_HQ_TIMEZONE = pytz.timezone('US/Pacific')
+SQL_TIME_FORMAT = '%Y-%m-%d %H:%M:%S'  # Time format favored by mysql
+
+
+def to_aware(local_time):
+    """ get a timezone-aware object from local time string(s).
+
+    Args:
+        local_time: string of local time in any valid non-timezone-aware ISO format
+            Time should be in Osmo HQ local time.
+            eg. ('2012-01-01 12:00:00', '2012-01-01 12:00', '2012-01-01')
+    Returns:
+        timezone-aware datetime object
+    """
+    time = datetime.datetime.fromisoformat(local_time)
+    return OSMO_HQ_TIMEZONE.localize(time)
+
+
+def to_utc_string(local_time):
```

Comment:
I wonder if all of these function names should have `osmo` in them, or something, to call out that they're hardcoded to Osmo HQ timezone.

That or maybe take the timezone as a param with Osmo as the default (though maybe that's just dangerous?)

Huh, wonder if Jona's going to be running any experiments from Boston?

Reply:
I'll follow up with Jona. i think this is the right level of specificity for now, we can add kwargs later if necessary. actually started with kwargs but backtracked because YAGNI